install_sct.bat

```diff
@@ -1,92 +1,93 @@
-@echo off
-rem Installation script for SCT on native Windows platforms
-rem Usage: install_sct.bat <version>
-rem e.g.
-rem        install_sct.bat 5.5
-
-rem Set git ref. If no git ref is specified when calling `install_sct.bat`, use a default instead.
-if [%1]==[] (
-  set git_ref=master
-) else (
-  set git_ref=%1
-)
-
-rem Go to user's home directory
-pushd %HOMEPATH%
-
-rem Check to see if we're going to git clone into an existing installation of SCT
-if exist spinalcordtoolbox\ (
-  echo ### Previous spinalcordtoolbox installation found at %HOMEPATH%\spinalcordtoolbox.
-  rem NB: The rmdir command will output 'spinalcordtoolbox\, Are you sure (Y/N)?', so we don't need our own Y/N prompt
-  rem     We also use "echo set /p=" here in order to make sure that Y/N text is output on the same line.
-  echo|set /p="### Continuing will overwrite the existing installation directory " || goto error

```

Comment:
I don't know what was the goal of the `|| goto error` on a simple echo but for me the problem was there, without it the bat file work like a charm

Reply:
If you have some knowledge about that @joshuacwnewton

Reply:
I don't think there was any big reasoning here -- I agree that it's fine to omit the `|| goto error` for the `echo`. :)